### DISCUSSION TITLE
Use external database

### DISCUSSION THREAD

**Discussion by adyanth** [ellite/Wallos#292] [Q&A]
I would like to use an external Postgres database instead of the embedded SQLite one (running in K8s, do not want the SQLite to get corrupted with a NFS volume mount). Is there a way to do this?

Edit: Is this what needs changing to support it? https://github.com/ellite/Wallos/blob/5fff7c59eb92147cfeb56d2b9bf595cd0800a8c3/includes/connect_endpoint.php

**Comment by ellite** [comment]
Hi Honestly not sure what would need to be changed. I believe most of the queries are SQLite specific, so those would need to also be changed.

**Reply by halkeye** [reply]
Would you be interested in someone contributing it? I'm not super familiar with modern php, but https://github.com/auraphp/Aura.SqlQuery/blob/3.x/docs/select.md seems pretty straight forward to swap out, and then its just a matter of using pdo different drivers.
